src/main/java/uk/gov/ons/ctp/response/casesvc/state/CaseSvcStateTransitionManagerFactory.java

```diff
@@ -1,5 +1,7 @@
 package uk.gov.ons.ctp.response.casesvc.state;
```

Comment:
:+1: All around this StateTransitionManager file is a lot clearer now

Reply:
Definitely easier to follow now in my opinion. Great work @benjefferies